packages/antd/src/templates/FieldTemplate/index.tsx

```diff
@@ -94,25 +94,21 @@ export default function FieldTemplate<
       uiSchema={uiSchema}
       registry={registry}
     >
-      {id === 'root' ? (
-        children
-      ) : (
```

Comment:
The reason the error didn't show up is because the example in #3608 had a String field as the root, and the error rendering is handled by `Form.Item`. Since there was no `Form.Item`, there was no visible error.